src/pages/gen2/build-a-backend/add-aws-services/rest-api/index.mdx

```diff
@@ -73,7 +73,7 @@ To initialize the REST API you need to configure Amplify with `Amplify.configure
 
 ```tsx title="pages/index.tsx"
 import { Amplify } from 'aws-amplify';
-import amplifyconfig from './src/amplifyconfiguration.json';
+import amplifyconfig from '../amplify_outputs.json';
 Amplify.configure(amplifyconfig):
```

Comment:
```suggestion
import amplifyoutputs from '../amplify_outputs.json';
Amplify.configure(amplifyoutputs):
```

Reply:
good call, updated all references to this import